prometheus-formula/prometheus/files/containers/alertmanager.container

```diff
@@ -0,0 +1,26 @@
+{%- set tls_enabled = salt['pillar.get']('prometheus:tls:enabled', False) %}
+{% set config = salt['pillar.get']('prometheus:alerting:alertmanager_config') %}
+{%- set entrypoint = ['/usr/bin/prometheus-alertmanager'] %}
+{%- if config %}
+{% do entrypoint.append('--config.file=' ~ config) %}
+{% endif -%}
+{%- if tls_enabled %}
+{% do entrypoint.append('--web.config.file=' ~ web_config_file) %}
+{%- endif -%}
+
+[Unit]
+Description=Alertmanager Container
+
+[Container]
+Label=app=alertmanager
+ContainerName=alertmanager
+Image=registry.opensuse.org/devel/bci/sle-15-sp6/containerfile/suse/alertmanager:0.26.0
```

Comment:
Do you want images from opensuse registry for SUMA as well?

Reply:
That is the point I wanted to discuss. For now I have just hard-coded them here.

Reply:
I checked that monitoring images are not being published to registry.suse.com. Please let me know if we want to have them there for our purposes.

Reply:
@admd @rjmateus probably for you

Reply:
From my point of view we cannot have opensuse images delivered for SUMA. We have a special SKU so we must have SUSE delivered images through registry.suse.com with special access control matching the needed SKU.

Reply:
Correct! We must use the images through registry.suse.com